PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/forms/AddNewClusterForm.java

```diff
@@ -58,12 +72,76 @@ protected void configureShell(Shell newShell) {
 		newShell.setText("Link New HDInsight Cluster");
 	}
 
+	// we cannot just override method 'helpPressed' since the method is defined as private in the parent class 'TrayDialog'
+	// so we disable the default help button and mock a button with different usage
+	protected void helpPressed() {
+		try {
+			PlatformUI.getWorkbench().getBrowserSupport().getExternalBrowser().openURL(new URL("https://go.microsoft.com/fwlink/?linkid=866472"));
```

Comment:
getExternalBrowser() may return `null`, you can use `Optional` here.

```
Optional.ofNullable(PlatformUI.getWorkbench())
    .map(IWorkbench::getBrowserSupport)
    .map(IWorkbenchBrowserSupport::getExternalBrowser)
    .ifPresent(browser -> browser.openURL(...))
```

Reply:
We have to catch PartInitException in map(IWorkbenchBrowserSupport::getExternalBrowser), which makes code messy.